os/binfmt/Kconfig

```diff
@@ -90,6 +90,16 @@ config OPTIMIZE_APP_RELOAD_TIME
 		the MPU requirements. Hence, using this option can result in a steep increase in the memory
 		requirement for this application.
 
+config BINFMT_SECTION_UNIFIED_MEMORY
+	bool "Allocate section memory as one chunk"
+	default n
```

Comment:
This is valid with reload optimization config enabled. Could you make a dependency with that?

Reply:
Done.